resources/newssite/news-nuxt/dist/404.html

```diff
@@ -3,6 +3,6 @@
 <head><meta charset="utf-8">
 <title>The Daily Broadcast</title>
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<meta name="description" content="A news site developed with Nuxt."><link rel="modulepreload" as="script" crossorigin href="./_nuxt/entry.0cc6ca39.js"><link rel="preload" as="style" href="./_nuxt/entry.762f5ea3.css"><link rel="prefetch" as="script" crossorigin href="./_nuxt/error-component.be57198b.js"><link rel="stylesheet" href="./_nuxt/entry.762f5ea3.css"></head>
-<body ><div id="__nuxt"></div><script type="application/json" id="__NUXT_DATA__" data-ssr="false">[{"_errors":1,"serverRendered":2,"data":3,"state":4},{},false,{},{}]</script><script>window.__NUXT__={};window.__NUXT__.config={public:{},app:{baseURL:"./",buildAssetsDir:"/_nuxt/",cdnURL:"./"}}</script><script type="module" src="./_nuxt/entry.0cc6ca39.js" crossorigin></script></body>
+<meta name="description" content="A news site developed with Nuxt."><link rel="modulepreload" as="script" crossorigin href="./_nuxt/entry.2b1d5601.js"><link rel="preload" as="style" href="./_nuxt/entry.b8cf5f77.css"><link rel="prefetch" as="style" href="./_nuxt/default.bd0e0714.css"><link rel="prefetch" as="script" crossorigin href="./_nuxt/default.3a4ed3f6.js"><link rel="prefetch" as="script" crossorigin href="./_nuxt/nuxt-link.cd8b4999.js"><link rel="prefetch" as="script" crossorigin href="./_nuxt/error-component.aa6b02dd.js"><link rel="stylesheet" href="./_nuxt/entry.b8cf5f77.css"></head>
```

Comment:
these additional prefetches are caused by using a layout (which is an async component). you should only ever use layouts if you have more than one - a single layout is better served by `app.vue` directly

Reply:
removed and created a wrapper component instead.